Mlem/MlemApp.swift

```diff
@@ -21,76 +22,28 @@ struct MlemApp: App {
                 .environmentObject(accountsTracker)
                 .onAppear {
                     URLCache.shared = AppConstants.urlCache
-                    loadAccounts()
 
                     setupAppShortcuts()
 
                     // set app theme to user preference
                     let windowScene =  UIApplication.shared.connectedScenes.first as? UIWindowScene
                     windowScene?.windows.first?.overrideUserInterfaceStyle = lightOrDarkMode
-                }
-        }
-        .onChange(of: accountsTracker.savedAccounts) { newValue in
-            do {
-                let encodedSavedAccounts: Data = try encodeForSaving(object: newValue)
 
-                do {
-                    try writeDataToFile(data: encodedSavedAccounts, fileURL: AppConstants.savedAccountsFilePath)
-                } catch let writingError {
-                    print("Failed while saving data to file: \(writingError)")
+                    let appearance = UITabBarAppearance()
```

Comment:
we should try and avoid use of `UIKit` like this since we're working in `SwiftUI`, what is the reason we need to add this?

Reply:
A weird bag that appears only on Pro Max phones where the tab view becomes fully transparent 

This tells the bar to keep the systemThinMaterial all the time (which looks like the default in iOS anyway)

Reply:
Is there a native SwiftUI way to do this?

Reply:
When I navigate back from Safari my tab bar is transparent for a moment (iPhone 13 Pro), that never used to happen - is there something we're doing that's causing this bug to surface?

Reply:
Example here:

https://github.com/mlemgroup/mlem/assets/5231793/d47e7ae2-729d-487f-9de9-daff1aaffecb

Reply:
Actually, if you notice the way we open, Safari is very dodgy, (there's some UIKit window manipulation)

But other than this, I have no idea what we might be doing to surface this,

About swiftUI to do it I'm not sure the swiftUI method applies as thoroughly as this one

Reply:
Yeah, I had changed that bit to stay within `SwiftUI` using a representable but David wasn't keen on it, can we raise a separate issue for this bug, perhaps if we go back to using a different approach for Safari it will alleviate it?

If we note it as an issue we can resolve it later, I don't think it's a blocker to merging

Reply:
raised as an issue here: https://github.com/mlemgroup/mlem/issues/231